hopp/simulation/technologies/pv/pv_plant.py

```diff
@@ -220,6 +250,18 @@ def footprint_area(self):
         area = num_modules * module_attribs['area']
         return  area
 
+    @property
+    def land_coverage_area(self):
```

Comment:
We talked about minor changes to the ``land_coverage_area`` property. I think it's fine to leave as-is and we can revisit it later OR to remove this property for now. What we discussed were
1) naming of ``land_coverage_area`` vs ``footprint_area``
2) I think the land-coverage area calc needs to include ground-coverage ratio (``self._system_model.SystemDesign.gcr``). 

These changes are not required for approval.

Reply:
How would you update the calculation to include the ground-coverage ratio?